### ISSUE TITLE
Relicensing from AGPL to MIT is illegal without consent from all past contributors.

### ISSUE THREAD

**Issue by tuxayo** [RainLoop/rainloop-webmail#2189]
Except for such trivial contributions that the authors can't claim copyright on them.

**In simple terms, what seems the most important is that other contributors must agree that RainLoop or a fork could be made non-libre/proprietary.** I'm not a lawyer but that's the most important difference between the two licenses.

Change was done in aa5d3fc89d67cb97b4b0f5c48ef32c39dd193975

@RainLoop The AGPL is a copyleft license, it's not compatible with MIT, so consent from all owners of the code must be obtained [1]. And there is no CLA that could have paved the way to relicensing.

Some references in case one of the contributors want to explore how DMCA Takedown works. (that's too early IMO before doing that, first step is dialog)
https://docs.github.com/en/site-policy/content-removal-policies/guide-to-submitting-a-dmca-takedown-notice
https://docs.github.com/en/site-policy/content-removal-policies/dmca-takedown-policy

[1] Which is very hard and some code would have to be rewritten for unreachable and unconsenting contributors.

cc part of the past contributors: @ervee @arturbonnett @skibbipl @hifihedgehog @insertc0in @sloomy @daliusm @strobeltobias @Techwolfy @esroyo @icedman21 @pierre-alain-b @plinss @KenBW2 @korukugashi @phunehehe @detochko @WesCossick @Rudloff @filipblaauw @alex-connor @yutianhot @yasselavila @erseco @kromonos @cz-vilda @zikeji @rhyswilliamsza @COLORLINK @almas @serghey-rodin @AndrewSav @nicolar @s4l1h @dvbmgr @james4321 @eddsstudio

**Comment by zikeji** [comment]
For what it's worth, I consent.

**Comment by phunehehe** [comment]
I would have had a problem with the switch if my contribution were more substantial. However it was almost trivial, and has been replaced anyway. So same as above, I consent, FWIW.

**Comment by icedman21** [comment]
FWIW, I consent

**Comment by plinss** [comment]
I consent

**Comment by skibbipl** [comment]
I thought project is dead. Anyway I also consent the license change.

**Comment by strobeltobias** [comment]
I consent the license change to MIT, too.

**Comment by kromonos** [comment]
I consent the license change to MIT, too.

**Comment by korukugashi** [comment]
I consent

**Comment by pierre-alain-b** [comment]
I consent.

That said, I wonder what the plan is with the project. I believe Rainloop is still widely used but my belief is that some forks are now more active than this initial repository.

**Comment by sloomy** [comment]
I consent too :)

**Comment by s4l1h** [comment]
I consent to the license changing to MIT.

**Comment by AndrewSav** [comment]
I consent to the license changing to MIT

**Comment by the-djmaze** [comment]
@tuxayo you can make a checkbox list to update how many agreed.
Use `- [ ]` like:

- [x] AndrewSav
- [ ] COLORLINK
- [ ] KenBW2
- [ ] Rudloff
- [ ] Techwolfy
- [ ] WesCossick
- [x] alex-connor
- [ ] almas
- [ ] arturbonnett
- [ ] cz-vilda
- [x] daliusm
- [ ] detochko
- [x] dvbmgr
- [ ] eddsstudio
- [x] erseco
- [ ] ervee
- [x] esroyo
- [ ] filipblaauw
- [ ] hifihedgehog
- [x] icedman21
- [ ] insertc0in
- [ ] james4321
- [x] korukugashi
- [x] kromonos
- [x] nicolar
- [x] phunehehe
- [x] pierre-alain-b
- [x] plinss
- [x] rhyswilliamsza
- [x] s4l1h
- [ ] serghey-rodin
- [x] skibbipl
- [x] sloomy
- [x] strobeltobias
- [x] the-djmaze
- [x] yasselavila
- [ ] yutianhot
- [x] zikeji

**Comment by erseco** [comment]
I consent to the license changing to MIT.

**Comment by nicolar** [comment]
I consent to the license changing to MIT.

**Comment by yasselavila** [comment]
I consent to the license changing to MIT.

**Comment by hansw-nlo** [comment]
Besides the question about the move being valid there are more issues. On the website, under licensing, it is still stated commercial customers are allowed to remove copyright. I would never accept that as a contributor of an open source project.

**Comment by AndrewSav** [comment]
@hansw-nlo as far as I understand, the change in license is because it is not longer sold - I could be wrong. Moreover that would be incompatible with MIT, which requires it to retain the notice (I'm assuming you mean the notice, not the actual copyright as it seems to make more sense)

**Comment by AndrewSav** [comment]
A statement from those in control of the repo would be nice to explain the community and the contributors what's happening and why.

**Comment by rhyswilliamsza** [comment]
Happy & I consent.

**Comment by alex-connor** [comment]
Great! I consent

**Comment by hansw-nlo** [comment]
> @hansw-nlo as far as I understand, the change in license is because it is not longer sold - I could be wrong. Moreover that would be incompatible with MIT, which requires it to retain the notice (I'm assuming you mean the notice, not the actual copyright as it seems to make more sense)

Indeed, the notice on the website itself. Afaik it is not legal to drop someone's copyright in either cases to be honest. It isn't bsd licensed

**Comment by nicolar** [comment]
Hello,
I consent to the license changing to MIT.

Regards,

Il giorno ven 30 set 2022 alle ore 10:25 the-djmaze <
***@***.***> ha scritto:

**Comment by daliusm** [comment]
I consent to the license changing to MIT

daliusm



2022-09-18, sk, 04:23 tuxayo ***@***.***> rašė:

**Comment by dvbmgr** [comment]
As my contribution was minor, I agree.

**Comment by esroyo** [comment]
I consent

El dimecres, 12 d’octubre de 2022, David Baumgartner (<
***@***.***>) va escriure:
> As my contribution was minor, I agree.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.<
https://ci4.googleusercontent.com/proxy/v6UeRJGOeQojXUQ7jEaH16FFrRW61tfIjNJkzpfeTJhrF0o-6TzFhlf4FdKbCaWQ0g4H8CjP2xbqCWFSLZgdEsHxh2ckUTZhWdckmJRV4UvPsay8pQaXzXxhULCKXuzVeJYdcPbandpPLNRh2S88JQeM8H1orE2k-64c7-3VLad0zJZGg1pg5N2MPUNfYhwQETPVl3qtsal1KSIJX-2-J4TpGqW9WLkGOUX7HLFtskigXf5wow=s0-d-e1-ft#https://github.com/notifications/beacon/AAH2NHQNBXVO5CNBZUNUJELWC2ZO5A5CNFSM6AAAAAAQPHLQL2WGG33NNVSW45C7OR4XAZNMJFZXG5LFINXW23LFNZ2KUY3PNVWWK3TUL5UWJTSMCBT7E.gif>Message
ID: ***@***.***>

-- 
Carles Escrig i Royo ~ simkin
